src/admin-dhis2/components/AdminDhis2DataElementForm/AdminDhis2DataElementForm.jsx

```diff
@@ -0,0 +1,146 @@
+import React, { useMemo, useEffect, useState } from 'react';
+import { toast } from 'react-toastify';
+
+import getService from '../../../react-components/utils/angular-utils';
+import { SearchSelect } from '../../../requisition-order-create/search-select'
+
+function AdminDhis2DataElementForm({onSubmit, onCancel, refetch, serverId, datasetId}) {
+
+    const indicatorTypeOptions = [
+        {name: 'Stock Management', value: 'Stock Management'},
+        {name: 'Requisition', value: 'Requisition'}
+    ];
+
+    const indicatorOptions = [
+        {name: 'Opening Balance', value: 'Opening Balance'},
+        {name: 'Received', value: 'Received'},
+        {name: 'Closing Balance', value: 'Closing Balance'},
+        {name: 'Total Negative Adjustments', value: 'Total Negative Adjustments'},
+        {name: 'Total Positive Adjustments', value: 'Total Positive Adjustments'}
+    ];
+
+    const [productOptions, setProductOptions] = useState([]);
+
+    const [providedName, setProvidedName] = useState("")
```

Comment:
please add semicolons at the end of lines: 24,25,26,27

Reply:
updated